device-types/Arctic-Wolf/awn1000-10g.yaml

```diff
@@ -0,0 +1,89 @@
+---
+manufacturer: Arctic Wolf
+model: AWN1000 10G
+slug: arctic-wolf-awn1000-10g
+description: 10G security sensor
+part_number: AWN1000 10G
+u_height: 1.0
+is_full_depth: true
```

Comment:
Hello, thank you for your contribution! In an attempt to have the most accurate and complete data we can, I have a few additional data points we would like you to add if possible.

1. Please add the `airflow` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not
2. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

If you have any questions, please let us know :)

Reply:
I was able to determine airflow is front-to-rear, however I could not locate information on the weight of the sensor.